dbt/printer.py

```diff
@@ -0,0 +1,214 @@
+
+from dbt.logger import GLOBAL_LOGGER as logger
+from dbt.utils import get_materialization, NodeType
+
+import colorama
+import time
+
+USE_COLORS = False
+
+def use_colors():
+    global USE_COLORS
+    USE_COLORS = True
+
+def get_timestamp():
+    return time.strftime("%H:%M:%S")
+
+
+def color(text, color_code):
+    if USE_COLORS:
+        return "{}{}{}".format(color_code, text, colorama.Style.RESET_ALL)
```

Comment:
@drewbanin is it possible to create an enumeration of available colors here, like:

```
RED_FOREGROUND = colorama.Fore.RED
GREEN_FOREGROUND = colorama.Fore.GREEN
YELLOW_FOREGROUND = colorama.Fore.YELLOW
...
```

and then reference those constants directly?

I'm thinking that would make it easy to override the colors when using dbt as a library